src/deepsparse/image_classification/pipelines.py

```diff
@@ -134,8 +145,17 @@ class properties into an inference ready onnx file to be compiled by the
 
         :return: file path to the ONNX file for the engine to compile
         """
+        model = Model(self.model_path)
```

Comment:
we need to re-work the logic here so passing in a local directory also works (this assumes zoo stub). should add a helper function (or re-work the existing unwrap zoo helper) to unwrap a string input to a deployment directory if it's a zoo stub (otherwise assume it is a deployment directory) and return it. then we can share logic for both local and zoo

Reply:
zoo or deployment path